ci/docker/runtime_functions.sh

```diff
@@ -73,6 +73,27 @@ build_jetson() {
     popd
 }
 
+build_armv6() {
+    set -ex
+    pushd .
+    cd /work/build
+    cmake\
+        -DUSE_CUDA=OFF\
+        -DUSE_OPENCV=OFF\
+        -DUSE_OPENMP=OFF\
```

Comment:
Could you elaborate why OPENMP is turned off?

Reply:
Brought it back